pkg/controller/razeedeployment/razeedeployment_controller.go

```diff
@@ -1436,15 +1436,17 @@ func (r *ReconcileRazeeDeployment) fullUninstall(
 		if err == nil {
 			for _, cr := range customResourceList.Items {
 				reqLogger.Info("Deleteing custom resource", "custom resource", cr)
-
-				err := r.client.Delete(context.TODO(), &cr, client.PropagationPolicy(deletePolicy))
+				err := r.client.Delete(context.TODO(), &cr)
 				if err != nil && !errors.IsNotFound(err) {
 					reqLogger.Error(err, "could not delete custom resource", "custom resource", cr)
 				}
```

Comment:
At this point we have deleted things, and we need to wait. This will requeue the event in 5 seconds.
```suggestion
				}

                              return reconcile.Result{requeueAfter: time.Second * 5}, nil
```